sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -8,27 +8,28 @@
     @submitted="submitted"
     @customEvent="customEvent"
   ></formio>
-  <v-row v-if="showNav">
-    <v-col>
-      <v-btn
-        color="primary"
-        v-show="!isFirstPage"
-        variant="outlined"
-        data-cy="previousSection"
-        @click="wizardGoPrevious"
-        >Previous section</v-btn
-      >
-    </v-col>
-    <v-col>
-      <v-btn
-        class="float-right"
-        color="primary"
-        v-show="!isLastPage"
-        @click="wizardGoNext"
-        >Next section</v-btn
-      >
-    </v-col>
-  </v-row>
+  <footer-buttons
+    justify="space-between"
+    :processing="processing"
+    @secondaryClick="wizardGoPrevious"
+    :showSecondaryButton="!isFirstPage"
+    secondaryLabel="Previous section"
+    class="mx-0"
+  >
+    <template #primary-buttons="{ disabled }">
+      <span>
```

Comment:
Is there a need to use the template for the primary button for this case?
I believe that the out-of-box properties are enough to do the personalization needed. If I am missing something please let me know.

Reply:
Ok. Removed the template. Thanks!